kong/plugins/acme/handler.lua

```diff
@@ -105,7 +105,7 @@ function ACMEHandler:certificate(conf)
   host = string.lower(host)
 
   if not check_domains(conf, host) then
-    kong.log.debug("ignoring because domain is not in whitelist")
+    kong.log.debug("ignoring because domain is not in whitelist:", host)
```

Comment:
Do you think we could make this `info` level as well? Feels like a useful operational message, and I suspect we'll drop debug messages at some point in the near future

Reply:
I agree with @Tieske here.